simple_test.py

```diff
@@ -0,0 +1,55 @@
+from typing import Dict, Optional, List
+from src.parser import parse_file
+from src.utils import get_good_idxs, fill_gaps, smooth_arr, compute_YP, convert_to_df, flatten_dict
+import numpy as np
+import argparse
+from src.visualizer import (
+    set_results_dir,
+)
+
+
+def main(filename: str, results_dir: str, vlines: Optional[List[float]] = None):
+    set_results_dir(results_dir)
+
+    """parse the file"""
+    data: Dict[str, np.ndarray or dict] = parse_file(filename)
+    print(data.keys())
+    # Convert to pandas dataframe
+    import pandas as pd
+    actors_data = {}
+    actors_data["TimeElapsed"] = data["TimeElapsed"]
+    actors_data["Actors"] = {}
+    actors_data["Actors"]["Location"] = data["Actors"]["Location"]
+  
+    
+
+    data_frame = convert_to_df(data)
+    actors_df = convert_to_df(actors_data)
+   
+    # Display the frame
+    from IPython.display import display
```

Comment:
This `display` variable is unused in the rest of the program. 

Also I'd recommend to move all your `import`s to the top of the file for better readability